go.mod

```diff
@@ -20,10 +20,10 @@ require (
 	github.com/olekukonko/tablewriter v0.0.5
 	github.com/prometheus/client_golang v1.14.0
 	github.com/prometheus/common v0.42.0
-	github.com/stretchr/testify v1.8.2
+	github.com/stretchr/testify v1.8.3
```

Comment:
Only packages under the `go.opentelemetry.io` were manually upgraded by me, the others are all modified by `go get -u`.